src/osd/OSD.cc

```diff
@@ -7376,7 +7376,7 @@ void OSD::ms_fast_dispatch(Message *m)
   if (m->otel_trace.IsValid()) {
     op->osd_parent_span = tracing::osd::tracer.add_span("op-request-created", m->otel_trace);
   } else {
-    op->osd_parent_span = tracing::osd::tracer.start_trace("op-request-created");
+    op->osd_parent_span = tracing::osd::tracer.start_trace("op-request-created", tracing::osd::tracer.is_enabled());
```

Comment:
this is a functional change. what is the reason for that?

Reply:
@ideepika ?

Reply:
sorry @yuvalif was traveling, I think this is not needed, I added this when I could not get OSD spans due to improper agent port

Reply:
removed this addition.